keyboards/torn/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega328p
+
+# Bootloader selection
+BOOTLOADER = USBasp
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = no        # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+ENCODER_ENABLE = yes        # Enable rotary encoder
+OLED_DRIVER_ENABLE = yes
+CUSTOM_MATRIX = lite
+
+SRC += \
+    i2c_master.c \
+    matrix.c \
+    mcp23018.c \
+    torn_encoder.c \
+    torn_oled.c
```

Comment:
```suggestion
SRC += matrix.c \
    torn_encoder.c \
    torn_oled.c
QUANTUM_LIB_SRC += i2c_master.c \
    mcp23018.c \
```

Reply:
```suggestion
SRC += \
    matrix.c \
    mcp23018.c \
    torn_encoder.c \
    torn_oled.c
QUANTUM_LIB_SRC += i2c_master.c
```

Reply:
Thanks, mcp23018.c is part of the keyboard source, but I've made the change to use QUANTUM_LIB_SRC for i2c_master.c.